src/govuk-prototype-kit/govuk-prototype-kit.config.unit.test.mjs

```diff
@@ -0,0 +1,163 @@
+import configFn from './govuk-prototype-kit.config.mjs'
+
+describe('GOV.UK Prototype Kit config', () => {
+  let config
+
+  beforeAll(async () => {
+    config = await configFn()
+  })
+
+  it('includes paths for assets, scripts, sass', () => {
+    expect(config.assets).toEqual([
+      '/govuk/assets'
+    ])
+
+    expect(config.sass).toEqual([
+      '/prototype-kit/init.scss'
+    ])
+
+    expect(config.scripts).toEqual([
+      '/govuk/all.js',
+      '/prototype-kit/init.js'
+    ])
+  })
+
+  describe('Nunjucks', () => {
+    it('includes macros list', () => {
+      expect(config.nunjucksMacros).toEqual([
```

Comment:
This doesn't quite feel like it's testing the right thing – if we added a new component and (somehow) the code we'd written meant that it _didn't_ get added to the list of macro imports, the tests would still pass. Conversely, as currently written this test will fail every time we add a new component.

Would it make sense to either test that the macros lists includes all current components, or stub out `getDirectories` with a mock return value and just test the transformations?

Reply:
Don't mind at all

But yep, new components will automatically break this test

I lifted and shifted the old `nunjucksMacros` array (plus missing components like header/footer) just to guarantee that nothing was missing and all the `componentNameToMacroName()` transforms exactly matched.

We use `expect.arrayContaining([])` for PostCSS plugins, shall I do that here?

It'll get stale and miss new components but at least tests won't fail

Reply:
Depending on what we decide to do re my other comment, I think I'd be inclined to stub out `getDirectories` and check that the transform does what we expect, and then check for the 'correctness' of the actual list of macros by testing the JSON output.